src/context/UserContext.jsx

```diff
@@ -0,0 +1,19 @@
+import { createContext, useContext, useState } from "react";
+
+const UserContext = createContext();
+
+const UserProvider = ({children}) => {
+    const [user, setUser] = useState('')
+    return <UserContext.Provider value={{user, setUser}}>{children}</UserContext.Provider>
+}
```

Comment:
Same here as the `EntryContext`, you want to double-check your formatting and make sure everything is aligned properly to ensure clear readability.